src/net_processing.cpp

```diff
@@ -148,6 +148,14 @@ namespace {
     std::unique_ptr<CRollingBloomFilter> recentRejects GUARDED_BY(cs_main);
     uint256 hashRecentRejectsChainTip GUARDED_BY(cs_main);
 
+    /**
+     * Filter for transactions that have been recently confirmed.
+     * We use this to avoid requesting transactions that have already been
+     * confirnmed.
+     */
```

Comment:
nit: I presume this makes some kind of doc tools unhappy cause they will add docs to the cs and not the var.

Reply:
Yeah, if you want to make a non-doxygen generic comment, use "/*" rather than "/**".

Reply:
Done.